<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>791</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Numero 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numero1"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Numero 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numero2"/>
     </item>
     <item>
      <widget class="QLabel" name="etiqueta_resultado">
       <property name="text">
        <string>Resultado</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boton_calcular">
       <property name="text">
        <string>Calcular</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
